Dockerfile

```diff
@@ -4,8 +4,8 @@ COPY --from=ghcr.io/astral-sh/uv:latest /uv /bin/uv
 # Copy the project into the image
 ADD . /mzai
```

Comment:
`ADD . /mzai`
should we change this path since we're no longer using any variation of it?